### PR TITLE
ENG-2567 removing repetitive invocation on opening step 1 in app tour

### PR DESCRIPTION
**Pull request by jeffgo10**
Best ways to test this fix is to use keycloak-activated AppBuilder:

 - open 'pages > designer > My homepage' then hard refresh the page. as soon as the app tour shows up, click 'Close' button and it will open the prompt with "yes" and "no" buttons. wait for it if it goes back to the first popup while designer data from the background loads. If it won't go back to the first step then it's fixed. :)
 - stress (cypress) test it. use `entando-test-automation` (i.e. `npm run browse-test`) and choose some scenario test file to run (make sure you run it with preview)

### CODE REVIEW

File: src/ui/app-tour/AppTour.js
Comment (jeffgo10) [reply src/ui/app-tour/AppTour.js:423]: yeah you are right @ichalagashvili . I didn't check until step 11 it does nothing when clicked. thanks for the heads up

Comment (jeffgo10) [reply src/ui/app-tour/AppTour.js:423]: hi @ichalagashvili I have added next step invocations on the steps that needed it since I removed the `goToStep` prop parameter above. I tested it and it works now. let me know your thoughts about it.

Comment (ichalagashvili) [reply src/ui/app-tour/AppTour.js:423]: awesome, thank you. approved!

**Review by ichalagashvili — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/1137#pullrequestreview-780761783" title="Approved">ichalagashvili</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [ichalagashvili]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
